var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -407,6 +408,11 @@ def autoreconf(self, spec, prefix):
         perl = which('perl')
         perl('autogen.pl')
 
+    def setup_build_environment(env):
```

Comment:
The first arg is self, then env

Reply:
@adamjstewart Of course. Sorry about that.